doc/user_guide.md

```diff
@@ -142,6 +154,21 @@ With this communication toolbox embedded devices can be used to listen and publi
 
 Currently the Xively Service supports a maximum message size of 128kb.
 
+
+### Secure File Transfer (SFT)
```

Comment:
What about this subsection title?
'### Secure File Transfer (SFT), Firmware Update (FWU)' ?